client/css/style.css

```diff
@@ -1598,6 +1598,12 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	display: none;
 }
 
+#settings [data-advanced] h2::after {
+	content: "advanced";
```

Comment:
That's gonna be painful to translate and you know it :)

Reply:
translations be damned?

Reply:
ill find another way 😞